src/System.Text.Primitives/System/Text/InvariantParser_signed.tt

```diff
@@ -0,0 +1,321 @@
+﻿<#@ template hostspecific="false" language="C#" #>
+<#@ output extension=".cs" #>
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
```

Comment:
do we manually generate the cs files from .tt or this will be done automatically through the build? may be adding some comment for how we generate the cs files from the tt files would be great.